JS/Lesson30/saffet/my-app/src/components/Ex9/index.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+import './index.css'
+
+export const Ex9 = () => {
+
+  const [backgroundColor, setBackgroundColor] = useState('bisque');
+
+  function changeBackgroundtoYellow(){
+    setBackgroundColor("yellow");
+  }
+
+  function changeBackgroundtoRed(){
+    setBackgroundColor("red");
+  }
+
+  return (
+    <>
+      <div className="btn-div" style={{backgroundColor: backgroundColor}}>
```

Comment:
It's always better to use a class that inline styling (`style={{backgroundColor: backgroundColor}}`) :)